src/region/RegionAnnotation.tsx

```diff
@@ -16,10 +17,32 @@ const RegionAnnotation = (props: Props, ref: React.Ref<HTMLAnchorElement>): JSX.
     const { isActive, shape, ...rest } = props;
     const { height, width, x, y } = shape;
     const className = classNames('ba-RegionAnnotation', { 'is-active': isActive });
+    const targetRef: React.Ref<SVGRectElement> = React.useRef(null);
+
+    React.useEffect(() => {
+        const scrollOptions: Options = {
+            behavior: 'smooth',
+            block: 'center',
+            inline: 'center',
+            scrollMode: 'if-needed',
+        };
+
+        if (isActive && targetRef.current) {
+            scrollIntoView(targetRef.current, scrollOptions);
+        }
+    }, [isActive]);
 
     return (
         <AnnotationTarget ref={ref} className={className} {...rest}>
-            <rect className="ba-RegionAnnotation-rect" height={height} rx={6} width={width} x={x} y={y} />
+            <rect
+                ref={targetRef}
```

Comment:
This is the target rect, not the target itself. Can we use the `ref` that's coming back from the target component above?